frontend/src/components/common/carousel/carousel.tsx

```diff
@@ -0,0 +1,84 @@
+import 'swiper/css/bundle';
+import './styles.css';
+import React, { useState } from 'react';
+
+import { FreeMode, Navigation, Thumbs } from 'swiper';
+import { Swiper, SwiperSlide } from 'swiper/react';
+
+import 'swiper/css';
+import 'swiper/css/free-mode';
+import 'swiper/css/navigation';
+import 'swiper/css/thumbs';
+import IconPark from './icon-park-solid_picture.svg';
+
+import type { Swiper as SwiperType } from 'swiper';
+
+export const Carousel = (): React.ReactElement => {
+  const [thumbsSwiper, setThumbsSwiper] = useState<SwiperType | null>(null);
+  const [currentImage, setCurrentImage] = useState(1);
+
+  const images = [
```

Comment:
Move it to a different file, because this component should use props to render the image

And if you try to pass images to this component, nothing will work